tests/e2e/src/pages/base.page.ts

```diff
@@ -280,4 +280,15 @@ export class BasePage {
       await expect(result).toBe(false);
     }
   }
+  async checkElementToContainText(elementType: string, value: string, checkText: string) {
```

Comment:
I suppose you can use existing method from cucumder by adding "partial string" verification

cucmber:
Then Element with "partial string" "No tokens was found for" should be "visible"

returnElementByType:

`else if (elementType === "partial string") {
      element = await this.getElementByPartialString(value);
    }`


`async getElementByPartialString(text: string) {
      element = await this.world.page?.locator(`//*[text()[contains(string(), '${text}')]]`).first();
      return await element;
    }`
    
    if everything if ok with this solution, you can delete the current one method.